Styles/style.css

```diff
@@ -91,27 +91,39 @@ body {
   margin: 0.5rem auto;
   max-width: 42rem;
   background-color: #323335a6;
-  box-shadow: 0 8px 32px 0 rgba(59, 59, 59, 0.37);
   backdrop-filter: blur(1px);
   -webkit-backdrop-filter: blur(1px);
   border: 2px solid rgba(255, 255, 255, 0.082);
   background-color: transparent;
+  border: 1px solid #2bc48a;
 }
 
 .hljs {
```

Comment:
The left-margin of the code blocks is causing the text to shift to right too much. Try checking it.

Reply:
Thank you @amanpandey04 for the review and suggestions.
fix colors of alerts, remove left-margin from code blocks,add pointer to nav-items, remove spinner toggle button as it was
not so smooth (thanks for pointing out). change mobile breakpoint to 767pxs,remove unnecessary comments